ISLE/isleapp.cpp

```diff
@@ -457,6 +491,63 @@ SDL_AppResult SDL_AppEvent(void* appstate, SDL_Event* event)
 		}
 		break;
 	}
+	case SDL_EVENT_JOYSTICK_BUTTON_DOWN: {
+		{
+			if (event->gbutton.button == ISLE_BUTTON_SPACE) {
+				if (InputManager()) {
+					InputManager()->QueueEvent(c_notificationKeyPress, SDLK_SPACE, 0, 0, SDLK_SPACE);
+				}
+			}
+			if (event->gbutton.button == ISLE_BUTTON_ESCAPE) {
+				if (InputManager()) {
+					InputManager()->QueueEvent(c_notificationKeyPress, SDLK_ESCAPE, 0, 0, SDLK_ESCAPE);
+				}
+			}
+		}
+		break;
+	}
+	case SDL_EVENT_JOYSTICK_AXIS_MOTION: {
+		if (event->gaxis.axis == ISLE_MOUSE_JOYSTICK_X) {
+			g_lastJoystickMouseX = ((float) event->gaxis.value) / SDL_JOYSTICK_AXIS_MAX * g_isle->GetMouseSensitivity();
+		}
+		else if (event->gaxis.axis == ISLE_MOUSE_JOYSTICK_Y) {
```

Comment:
You've defined both `ISLE_MOUSE_JOYSTICK_X` and `ISLE_MOUSE_JOYSTICK_Y` as 0, which is `SDL_GAMEPAD_AXIS_LEFTX`.

That's unlikely to be correct?

```suggestion
		else if (event->gaxis.axis == SDL_GAMEPAD_AXIS_LEFTX) {
```